lib/perl/Genome/VariantReporting/Framework/Plan/ExpertPlan.pm

```diff
@@ -21,21 +21,22 @@ sub category {
 
 sub needed_translations {
     my $self = shift;
-    return Set::Scalar->new(map {$self->adaptor_params->{$_}}
-        $self->get_class->adaptor_class->required_translated_input_names);
+    my $needed = Set::Scalar->new(map {$self->run_params->{$_}} $self->get_class->run_class->required_translated_input_names);
+    $needed->insert(map {@{$self->run_params->{$_}}} $self->get_class->run_class->required_translated_is_many_input_names);
```

Comment:
I just realized that combining these doesn't make much sense because we need to handle them differently in terms of dereferencing the run_params call.